ktor-features/ktor-websockets/jvm/test/io/ktor/tests/websocket/WebSocketEngineSuite.kt

```diff
@@ -604,7 +604,7 @@ abstract class WebSocketEngineSuite<TEngine : ApplicationEngine, TConfiguration
     }
 
     @Test
-    fun testClientClosingFirst() {
+    open fun testClientClosingFirst() {
```

Comment:
Why it became open?

Reply:
To override and ignore in child `TomcatWebSocketTest`

Reply:
Then it probably shouldn't inherit `WebSocketEngineSuite` anymore

Reply:
Then it probably shouldn't inherit `WebSocketEngineSuite` anymore